terraso_backend/apps/storage/services.py

```diff
@@ -0,0 +1,11 @@
+from .s3 import ProfileImageStorage
+
+
+class ProfileImageService:
+    def upload(self, user_id, image):
+        profile_image_storage = ProfileImageStorage()
+        profile_image_storage.save(user_id, image)
```

Comment:
Doesn't this call raise any exception? What happens when there's a connectivity issue, for example?

Reply:
Good point, I will check that